### PR TITLE
Feature/search movies by name function

### PR DESCRIPTION
**Pull request by lukisko**
finished filter of movies - requirement 4. At least I hope so.

### CODE REVIEW

**Comment by lukisko** [comment]
Fixed the above mentioned problem

File: Ui/Http/Pages/Index.razor
Comment (KlausBenislavskis) [inline Ui/Http/Pages/Index.razor:1]: some repetitive stuff perhaps can be written cleaner

**Review by KlausBenislavskis — COMMENTED** [review]
Nice work

File: Ui/Http/Pages/Index.razor [hunk 1]
```diff
@@ -1,44 +1,102 @@
...
 
 @code {
+    private int minYear { get; set; } = 1800;
+    private int maxYear { get; set; } = System.DateTime.Now.Year;
+    private bool showFilterBox = false;
+    private Func<int, Task<MovieList>>? moviesToDisplay = null;
+    private GenreList genreList = new() { Genres = new() };
+    private int selectedGenreId { get; set; } = emptyGenre.Id;
+    private static readonly Genre emptyGenre = new() { Id = -1, Name = "---" };
+    private Person? filterCrew { get; set; }
```
Comment (lukisko) [inline Ui/Http/Pages/Index.razor:80]: Not sure if these should stay here as Andrej moved them to Ui/Http/Shared/Filter/FilterMovies.razor

**Pull request merged**